src/main/java/Duke/EmptyDescriptionException.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+
+/**
+ * Exception for tasks without a description.
+ */
+public class EmptyDescriptionException extends Exception {
+    public EmptyDescriptionException(String message) {
+        super(message);
+    }
+
+    @Override
+    public String toString() {
+        return getMessage();
+    }
+}
```

Comment:
I noticed that some of your files do not end with a new line. I think it is better to do so to make conflict resolution easier.